lib/services/auth/filters/getScopeFilter.js

```diff
@@ -11,6 +11,7 @@ import getUserScopeFilter from 'lib/services/auth/modelFilters/user';
 import getDownloadScopeFilter from 'lib/services/auth/modelFilters/download';
 import getExportScopeFilter from 'lib/services/auth/modelFilters/export';
 import getQueryScopeFilter from 'lib/services/auth/modelFilters/query';
+import getFullActivities from 'lib/services/auth/modelFilters/fullActivities';
```

Comment:
I think we should rename this to `getFullActivitiesFilter` to match the other names for consistency.